pinakes/main/migrations/0043_alter_source_refresh_state.py

```diff
@@ -16,7 +16,7 @@ class Migration(migrations.Migration):
             field=models.CharField(
                 choices=[
                     ("Done", "Done"),
-                    ("InProgress", "Inprogress"),
+                    ("InProgress", "In Progress"),
```

Comment:
@hsong-rh Since this is going to be displayed to the end user should it have
```
from django.utils.translation import gettext_lazy as _
...
("InProgress", _("In Progress")),
```

Reply:
@mkanoor Isn't the displayed value only used by drf-ui? Since this is migration, is it still meaningful if later the user changes the language?

Reply:
@bzwei it gets displayed in the Platform Summary Screen

<img width="1290" alt="Screen Shot 2022-04-26 at 5 23 10 PM" src="https://user-images.githubusercontent.com/6452699/165394755-61730f95-1c24-4d41-acb0-efb7a521d804.png">

Reply:
@hsong-rh @bzwei Do we need a migration here, can we just have the enum/choices directly defined in the model as a static and then load up the text based on a constant value

Reply:
I believe this migration is auto generated. Every time when we modify the enums in the model, a new migration is generated. 
Looking at the model, the state is a char field, not integer. So the actual text "In progress" is stored. 
@hsong-rh did you manually edit the migration file to change `InProgress` to `In progress`? I see the model constant value is `InProgress`.

Reply:
@bzwei Everything here is auto generated

Reply:
@hsong-rh 

```
from django.utils.translation import gettext_noop


.....

 ("InProgress", gettext_noop("In Progress")),

```

The gettext_noop allows for strings to be picked up for translation and the gettext_lazy will pick up the correct string at runtime.

Reply:
@hsong-rh Sorry my mistake I think we can. leave the migrations out of this, these changes need to be in our model file